### DISCUSSION TITLE
An error occur please see console for more information

### DISCUSSION THREAD

**Discussion by yiwaixiao** [Zain-ul-din/whatsapp-ai-bot#17] [General]
Thank you for providing the whatsapp-ai-bot. I tested it and it's a great application. However, I encountered the following issue during the trial. Whenever I engage in a few rounds of chat, I receive the following error. Is this a bug in the program or is there something that needs to be adjusted? Please advise.

✖ CustomModel request fail | An error occur, at CustomModel.ts sendMessage(prompt, msg) err: Error: OpenAI error 400: {
"error": {
"message": "This model's maximum context length is 4097 tokens. However, you requested 4246 tokens (1246 in the messages, 3000 in the completion). Please reduce the length of the messages or completion.",
"type": "invalid_request_error",
"param": "messages",
"code": "context_length_exceeded"
}
}

**Comment by Zain-ul-din** [comment]
[Check this issue for answer](https://github.com/Zain-ul-din/whatsapp-ai-bot/issues/16)
